tests/app/src/Controller/GetDocumentController.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller;
+
+use Pimcore\Model\Document;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Attribute\Cache;
+
+final class GetDocumentController
```

Comment:
```suggestion
#[AsController]
final class GetDocumentController
```

Also, it seems to work without it, though 🤔